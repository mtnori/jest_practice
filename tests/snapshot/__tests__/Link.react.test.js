import React from 'react';
import renderer from 'react-test-renderer';
import Link from '../Link.react';

it('正しくレンダリングされる', () => {
  const tree = renderer
    .create(<Link page="http://www.instagram.com">Instagram</Link>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
