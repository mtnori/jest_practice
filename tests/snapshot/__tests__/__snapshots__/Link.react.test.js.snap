// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`正しくレンダリングされる 1`] = `
<a
  className="normal"
  href="http://www.instagram.com"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  Instagram
</a>
`;
